### PR TITLE
Use Cobra command line tool

### PR DESCRIPTION
**Pull request by sukeesh**
Use Cobra command line tool

### CODE REVIEW

File: cmd/convert.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	"github.com/sukeesh/markitdown-go/pkg/pdfconverter"
+)
+
+var convertCmd = &cobra.Command{
+	Use:   "convert [pdf file]",
+	Short: "Convert a PDF file to Markdown",
+	Long: `Convert a specified PDF file to a Markdown (.md) file, extracting text and images.
+
+Example:
+  pdf2md convert sample.pdf -o result.md -a images`,
```
Comment (re1nth) [inline cmd/convert.go:18]: create .md file before conversion ?

Comment (sukeesh) [reply cmd/convert.go:18]: Did you try? This code creates the .md file @re1nth

File: cmd/convert.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		assetsDir, _ := cmd.Flags().GetString("assets")
+
+		// Validate input file
+		absInputPath, err := filepath.Abs(inputPath)
+		if err != nil {
+			log.Fatalf("Invalid input path: %v", err)
+		}
+
+		// Perform conversion
+		err = pdfconverter.ConvertPDFToMarkdown(absInputPath, outputMarkdown, assetsDir)
```
Comment (re1nth) [inline cmd/convert.go:34]: create a new md file if not provided

File: cmd/root.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
```
Comment (re1nth) [inline cmd/root.go:27]: explain this ?

File: output/output.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+
+![Image1](/Users/sukeesh/go/src/github.com/sukeesh/markitdown-go/assets/sodapdf-converted_1_Im1.png)
+
+![Image 2](/Users/sukeesh/go/src/github.com/sukeesh/markitdown-go/assets/sodapdf-converted_2_Im2.png)
```
Comment (re1nth) [inline output/output.md]: exclude this file ?

**Review by re1nth — APPROVED** [review]

File: output/output.md (hunk 4 above)
Comment (sukeesh) [reply output/output.md]: Done.

**Pull request merged**